src/node_contextify.cc

```diff
@@ -205,7 +208,11 @@ class ContextifyContext {
 
     Local<Context> ctx = Context::New(env->isolate(), nullptr, object_template);
 
-    CHECK(!ctx.IsEmpty());
+    if (ctx.IsEmpty()) {
+      env->ThrowError("Could not instantiate context");
+      return ctx;
```

Comment:
Not wrong but `return Local<Context>()` makes it clear without, ah, context that we're returning an empty handle.